etc/ci/buildbot_steps.yml

```diff
@@ -156,6 +176,7 @@ windows-msvc-dev:
   - mach.bat test-stylo
 
 windows-msvc-nightly:
+  - mach.bat clean-cargo-cache --keep 3 --force --custom-path
```

Comment:
Since #19395 has merged, let's make the --custom-path behaviour the default behaviour and remove the argument entirely from the mach command implementation.

Reply:
@jdm Done